### DISCUSSION TITLE
Fargate: customize requested vCPU and memory resources

### DISCUSSION THREAD

**Discussion by m1x0n** [artilleryio/artillery#2216] [Artillery]
Is there a way or do you plan to add some options to configure fargate instance resources (vCPU, memory)?

**Comment by hassy** [comment]
hi @m1x0n 👋 yes it's possible, just pushing a quick update to the docs

**Reply by hassy** [reply]
added an example here: https://www.artillery.io/docs/reference/cli/run-fargate#examples

**Reply by m1x0n** [reply]
Thank you! The last time I tried `run-fargate` defaults were 4vCPU and 8Gb Memory.

**Reply by hassy** [reply]
Ah yes I'll update the example. If you're running Playwright tests I've found that cranking up the memory to 12GB produces much more reliable results.

**Reply by m1x0n** [reply]
Ah, got it.

**Comment by garylparas** [comment]
Seems like the max vCPU it can support is 4096? I got this error when attempting to use 8192 vCPU.
Error: Unsupported cpu override for Fargate. Must be one of: 256, 512, 1024, 2048, 4096

Same with memory the 4 vCPU can only support up to 12288 memory. It errors out when I set up the memory above 12288.
Fargate memory override for cpu = 4096 must be one of: 8192, 9216, 10240, 11264, 12288

Not really sure if this is an Artillery limitation because in the documentation 4 vCPU can support up to 30GB memory.
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html

@hassy , hope you can enlighten us. Thank you
